common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -63,9 +63,29 @@ public class CommonValue {
             "    },\n" +
             "    \""+MLModelGroup.LATEST_VERSION_FIELD+"\": {\n" +
             "      \"type\": \"integer\"\n" +
-            "    }\n" +
+            "    },\n" +
```

Comment:
Seems tags mapping is missing

Reply:
Yes, had to remove tags and backend_roles mapping because they were throwing exception of incorrect mappings. Was working on them. Will add them back in next commit.